From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 21 Nov 2012 12:46:40 -0800
Patch-mainline: v3.8-rc1
Subject: hv: hv_balloon: mark a function static
Git-commit: 989623c7d687d3996d5676a6e1474dc7f01bf158
References: fate#314663

This resolves the following sparse warning:

drivers/hv/hv_balloon.c:548:6: sparse: symbol 'free_balloon_pages' was not declared. Should it be static?

Reported-by: Xie ChanglongX <changlongx.xie@intel.com>
Cc: K. Y. Srinivasan <kys@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: <ohering@suse.de>
---
 drivers/hv/hv_balloon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/hv/hv_balloon.c b/drivers/hv/hv_balloon.c
index bbc4973..f6c0011 100644
--- a/drivers/hv/hv_balloon.c
+++ b/drivers/hv/hv_balloon.c
@@ -545,7 +545,7 @@ static void post_status(struct hv_dynmem_device *dm)
 
 
 
-void free_balloon_pages(struct hv_dynmem_device *dm,
+static void free_balloon_pages(struct hv_dynmem_device *dm,
 			 union dm_mem_page_range *range_array)
 {
 	int num_pages = range_array->finfo.page_cnt;
